docs/installing-open-mpi/sysadmins.rst

```diff
@@ -0,0 +1,167 @@
+Advice for System Administrators
+================================
+
+Several members of the Open MPI team have strong system
+administrator backgrounds; we recognize the value of having software
+that is friendly to system administrators.  Here are some of the reasons
+that Open MPI is attractive for system administrators:
+
+* Simple, standards-based installation
+* Reduction of the number of MPI installations
+* Ability to set system-level and user-level parameters
+* Scriptable information sources about the Open MPI installation
+
+See the rest of this section for more details.
+
+Setting Global MCA Parameters
+-----------------------------
+
+MCA parameters are a way to tweak Open MPI's behavior at
+run-time.  For example, MCA parameters can specify:
+
+* Which interconnect networks to use
+* Which interconnect networks *not* to use
+* The size difference between eager sends and rendezvous protocol sends
+* How many registered buffers to pre-pin (e.g., for InfiniBand)
+* The size of the pre-pinned registered buffers
+* ...etc.
+
+It can be quite valuable for a system administrator to play with such
+values a bit and find an "optimal" setting for a particular
+operating environment.  These values can then be set in a global text
+file that all users will, by default, inherit when they run Open MPI
+jobs.
+
+For example, say that you have a cluster with 2 ethernet networks |mdash|
+one for NFS and other system-level operations, and one for MPI jobs.
+The system administrator can tell Open MPI to not use the NFS TCP
+network at a system level, such that when users invoke ``mpirun`` or
+``mpiexec`` to launch their jobs, they will automatically only be using
+the network meant for MPI jobs.
+
+:ref:`See how to set MCA params
+<label-running-setting-mca-param-values>` for information on how to
+set global MCA parameters.
+
+Setting MCA Parameters for a Global Open MPI installation
+---------------------------------------------------------
+
+Which global MCA parameters to set depends on both your specific
+parallel setup and the applications that typically run there.
+
+The best thing to do is to use the ``ompi_info`` command to see what
+parameters are available and relevant to you.  Specifically,
+``ompi_info`` can be used to show all the parameters that are available
+for each plug-in.  Two common places that system administrators like
+to tweak are:
+
+* *Only allow specific networks:* Say you have a cluster with a
+  high-speed interconnect (such as InfiniBand) and a
+  low-speed ethernet network (e.g., 1Gpbps).  The high-speed network is intended for MPI jobs;
+  the control network is intended for NFS and other
+  administrative-level tasks.  In this case, you can simply turn off Open
+  MPI's TCP support.  The ``btl`` framework contains Open MPI's network
```

Comment:
This section is kind of outdated. Outside of BTL's, we have MTL's and UCX has its own pml too.

Reply:
It's an example of how to configure the Open MPI installation, so it doesn't have to mention all the options available.